cmd/swarm/main.go

```diff
@@ -440,7 +440,7 @@ func getPassPhrase(prompt string, i int, passwords []string) string {
 	return password
 }
 
-func injectBootnodes(srv *p2p.Server, nodes []string) {
+func injectBootnodes(srv p2p.Server, nodes []string) {
```

Comment:
so you changed p2p.Server to an interface which is implemented by p2p.server (rlpx server) and also adapters.SimNode? 
genious